tap_decentraland_api/places_streams.py

```diff
@@ -0,0 +1,117 @@
+"""Events stream class for tap-decentraland-api."""
+
+import logging
+import requests
+from pathlib import Path
+from typing import Any, Dict, Optional, Union, List, Iterable
+from singer_sdk.streams import RESTStream
+
+from singer_sdk.typing import (
+    ArrayType,
+    BooleanType,
+    DateTimeType,
+    IntegerType,
+    NumberType,
+    ObjectType,
+    PropertiesList,
+    Property,
+    StringType,
+)
+
+SCHEMAS_DIR = Path(__file__).parent / Path("./schemas")
+
+class PlacesStream(RESTStream):
+
+    RESULTS_PER_PAGE = 100
+
+    @property
+    def url_base(self) -> str:
+        """Return the API URL root, configurable via tap settings."""
+        return self.config["places_api_url"]
+
+    def parse_response(self, response) -> Iterable[dict]:
+        """Parse the response and return an iterator of result rows."""
+        data = response.json().get("data")
+        try:
+            for row in data:                
+                yield row
+        except Exception as err:
+            self.logger.warn(f"(stream: {self.name}) Problem with response: {data}")
+            raise err
+
+    def get_next_page_token(
```

Comment:
Are you sure this API is paged? It looks like it returns all places instead from what i'm seeing